src/pages/profile/index.ts

```diff
@@ -3,23 +3,43 @@ import './profile.scss';
 import Component from '../../utils/Component';
 import Button from '../../components/button/index';
 import Input from '../../components/input';
+import Popup from '../../components/popup';
 import ProfileItem from '../../components/profileItem/index';
 import imgUrl from '../../assets/icons/iconProfile.svg'
+import closeIcon from '../../assets/icons/close.svg'
 import dataIcon from '../../assets/icons/changeDataIcon.svg';
-import { validateInput } from '../../utils/validation'
+import { validateInput } from '../../utils/validation';
+import store, { StoreEvents } from '../../utils/Store';
+import AuthController from '../../controllers/AuthController';
+import UserController from '../../controllers/UserController';
+import Router from '../../utils/Router';
 
 type ProfileProps = {
     src: string,
     title: string,
     dataList: object[],
     buttons: object[],
+    popup: object,
+    events: object,
     attr: object
 };
 
 class Profile extends Component<ProfileProps> {
 
     constructor(props: ProfileProps) {
         super('section', props);
+        store.on(StoreEvents.Updated, () => {
+            emailInput.setProps({placeholder: (store.getState().currentUser as any).email});
+            loginInput.setProps({placeholder: (store.getState().currentUser as any).login});
+            firstNameInput.setProps({placeholder: (store.getState().currentUser as any).first_name});
+            secondNameInput.setProps({placeholder: (store.getState().currentUser as any).second_name});
+            phoneInput.setProps({placeholder: (store.getState().currentUser as any).phone});
```

Comment:
добавь
```suggestion
                        emailInput.setProps({
                placeholder: (store.getState().currentUser as any).email
            });
            loginInput.setProps({
                placeholder: (store.getState().currentUser as any).login
            });
            firstNameInput.setProps({
                placeholder: (store.getState().currentUser as any).first_name
            });
            secondNameInput.setProps({
                placeholder: (store.getState().currentUser as any).second_name
            });
            phoneInput.setProps({
                placeholder: (store.getState().currentUser as any).phone
            });
```

Так будет читаемее выглядеть